services/wfm/src/main/java/org/openkilda/wfm/topology/nbworker/bolts/LinkOperationsBolt.java

```diff
@@ -225,6 +232,44 @@ private LinkPropsResponse dropLinkProps(LinkPropsDrop request) {
         }
     }
 
+    private LinkProps deleteLinkProps(LinkProps linkPropsToDrop) {
+        LinkPropsRepository linkPropsRepository = repositoryFactory.createLinkPropsRepository();
+        IslRepository islRepository = repositoryFactory.createIslRepository();
```

Comment:
You create two repos on each request. I think it's better to make these repos static and transient and initialize them in `init()` method. And reuse them in other methods

P.S. Honestly, all code from `deleteLinkProps` and `createOrUpdateProps` must be in class LinkOperationsService